client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -154,7 +154,7 @@ class CustomContentTypes extends Component {
 
 	renderPortfolioSettings() {
 		const { translate } = this.props;
-		const fieldLabel = translate( 'Portfolio Projects' );
+		const fieldLabel = translate( 'Portfolio projects' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.